recipes/antarna/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: antarna
+  version: "2.0.1.1"
+
+source:
+  git_url: https://github.com/RobertKleinkauf/antarna.git
```

Comment:
Can you please use a tarball and not the git master. For example this one is your latest commit: https://github.com/RobertKleinkauf/antarna/archive/4811c5e92b839362dff1ac472cb37b792a40a690.tar.gz